csharp/SparkCLR.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.SparkCLR</id>
+    <version>1.0.0</version>
+    <authors>Kaarthik Sivashanmugam, Renyi Xiong</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>https://github.com/Microsoft/SparkCLR/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/SparkCLR</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
```

Comment:
I will get back to this PR after learning more about this tag in the context of this project